demisto_sdk/main.py

```diff
@@ -272,6 +273,23 @@ def format(file_type, **kwargs):
     return 1
 
 
+@main.command(name="init", short_help="Initiate a new Pack, Integration or Script.")
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-n", "--name", help="The name of the object you want to create")
+@click.option(
+    "-o", "--outdir", help="The output dir to write the object into")
+@click.option(
+    '--integration', is_flag=True, help="Create an Integration")
+@click.option(
+    '--script', is_flag=True, help="Create a script")
+def init(**kwargs):
```

Comment:
lets add a --empty option to start things from a clean slate

Reply:
what do you mean? like an empty .py file and .yml files? why would you use this instead of simply deleting the HelloWorld contents?